src/Decoder/CallbackResponseDecoder.php

```diff
@@ -30,12 +30,9 @@ public function __construct(callable $supports, callable $decode)
         $this->decode = $decode;
     }
 
-    /**
-     * @param mixed $response
-     */
-    public function supports($response): bool
+    public function supports($response, $initiator): bool
```

Comment:
I guess that `$initiator` wouldn`t be necessary all the time. So I think it would be better to make it `null` by default 🤔 ...

Reply:
Well `null` as a default doesn't mean it is not necessary all the time, it means you may not pass an initiator when you call a supports method - which you don't want to allow. But if you don't have an initiator, you can pass a `null`.